sdk/loadtestservice/loadtestservice/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist-esm",
+    "esModuleInterop": true,
+    "paths": {
+      "@azure-rest/loadtestservice": ["./src/index"]
+    },
+    "types": []
+  },
+  "include": ["src/**/*.ts", "./test/**/*.ts", "samples-dev/**/*.ts"]
+}
```

Comment:
@joheredi is this auto-generated? is it intentional to not extend from the root-level tsconfig?

Reply:
Looking at it, this doesn't seem to be generated, we need to add 

```json
"extends": "../../../tsconfig.package",
```